### DISCUSSION TITLE
现在指定机器人切换目标不好使

### DISCUSSION THREAD

**Discussion by JoshuaWuP** [mod-playerbots/mod-playerbots#206] [Q&A]
机器人不按照玩家指定的攻击目标进行攻击，并且melee包含了tank，即指挥melee时候tank也会动作，另外当有俩个以上玩家召唤的机器人在团队时，指令会受到混淆，建议机器人只听随所属玩家的指令。

**Comment by cj-br** [comment]
骷髅标记偶尔会起作用

**Comment by liyunfan1223** [comment]
https://github.com/liyunfan1223/mod-playerbots/commit/e16e659df84432dbf21c51df1296e5f69f4463eb Attack command and skull marker will now correctly cause the bot to focus fire.
